components/drawer/index.tsx

```diff
@@ -15,14 +19,16 @@ const FormDrawer = () => {
             anchor="right"
             open={showing}
             PaperProps={{ style: { width: '50%' } }}>
-            <StyledDrawerHeader>
-                <span>{title}</span>
-                <Close
-                    className="close-icon"
-                    onClick={() => dispatch(closeDrawer())}
-                />
-            </StyledDrawerHeader>
-            {content}
+            <StyledDrawerContent>
+                <StyledDrawerHeader>
+                    <span>{title}</span>
+                    <Close
+                        className="close-icon"
+                        onClick={() => dispatch(closeDrawer())}
+                    />
+                </StyledDrawerHeader>
+                {content === TESTIMONIAL ? <TestimonialForm /> : '{ content }'}
```

Comment:
Good step forward, but i think we should keep pushing you here 😄

Switch this from a inline ternary to a `switch` block outside of the return. 
1. Create a function called `renderDrawerBody`
2. Put switch statement in the function above
3. Each case should be the content types, TESTIMONIAL, CONTACT, etc
4. Call function in return